### ISSUE TITLE
Fix reply to tweet text + img

### ISSUE THREAD

**Issue by StephanAkkerman** [StephanAkkerman/fintwit-bot#310]
Fix the text + url (link to reply instead)

![image](https://user-images.githubusercontent.com/45365128/206421711-04e9ef36-a397-4dc1-a5de-bb00c201da2c.png)
![image](https://user-images.githubusercontent.com/45365128/206421727-f0bd006a-0bca-41d4-9609-5fac77254fd9.png)
https://twitter.com/HsakaTrades/status/1600729171789254656

**Comment by StephanAkkerman** [comment]
https://twitter.com/HsakaTrades/status/1757095807801602092

![image](https://github.com/StephanAkkerman/fintwit-bot/assets/45365128/93518d2f-7a5f-4b97-8d5b-bd1fb67339b9)

**Comment by StephanAkkerman** [comment]
```json
{'entryType': 'TimelineTimelineItem', '__typename': 'TimelineTimelineItem', 'itemContent': {'itemType': 'TimelineTweet', '__typename': 'TimelineTweet', 'tweet_results': {'result': {'__typename': 'Tweet', 'rest_id': '1794291231473750116', 'core': {'user_results': {'result': {'__typename': 'User', 'id': 'VXNlcjo5MTgxMzgyNTM2MTc3OTA5NzY=', 'rest_id': '918138253617790976', 'affiliates_highlighted_label': {}, 'has_graduated_access': True, 'is_blue_verified': True, 'profile_image_shape': 'Circle', 'legacy': {'following': True, 'can_dm': False, 'can_media_tag': False, 'created_at': 'Wed Oct 11 15:36:32 +0000 2017', 'default_profile': False, 'default_profile_image': False, 'description': 'Fulltime Crypto Trader & Investor 📈 | Tweets are not financial advice | \nTG: https://t.co/i0wbPovD2j | YT: https://t.co/9mWXVagLLk |', 'entities': {'description': {'urls': [{'display_url': 't.me/DaanCryptoTG', 'expanded_url': 'http://t.me/DaanCryptoTG', 'url': 'https://t.co/i0wbPovD2j', 'indices': [77, 100]}, {'display_url': 'youtube.com/@daancrypto', 'expanded_url': 'http://www.youtube.com/@daancrypto', 'url': 'https://t.co/9mWXVagLLk', 'indices': [107, 130]}]}, 'url': {'urls': [{'display_url': 'partner.bybit.com/b/daancrypto', 'expanded_url': 'https://partner.bybit.com/b/daancrypto', 'url': 'https://t.co/I8pClGdemh', 'indices': [0, 23]}]}}, 'fast_followers_count': 0, 'favourites_count': 111012, 'followers_count': 372092, 'friends_count': 455, 'has_custom_timelines': True, 'is_translator': False, 'listed_count': 4911, 'location': 'collaborations@daancrypto.com', 'media_count': 9313, 'name': 'Daan Crypto Trades', 'normal_followers_count': 372092, 'pinned_tweet_ids_str': ['1403777391147851778'], 'possibly_sensitive': False, 'profile_banner_url': 'https://pbs.twimg.com/profile_banners/918138253617790976/1714510854', 'profile_image_url_https': 'https://pbs.twimg.com/profile_images/1593241078035279874/7ousc-BT_normal.jpg', 'profile_interstitial_type': '', 'screen_name': 'DaanCrypto', 'statuses_count': 42011, 'translator_type': 'none', 'url': 'https://t.co/I8pClGdemh', 'verified': False, 'want_retweets': True, 'withheld_in_countries': []}, 'professional': {'rest_id': '1590437698577342464', 'professional_type': 'Creator', 'category': []}}}}, 'unmention_data': {}, 'edit_control': {'initial_tweet_id': '1794290924048052457', 'edit_control_initial': {'edit_tweet_ids': ['1794290924048052457', '1794291231473750116'], 'editable_until_msecs': '1716630867000', 'is_edit_eligible': True, 'edits_remaining': '4'}}, 'previous_counts': {'bookmark_count': 0, 'favorite_count': 7, 'quote_count': 0, 'reply_count': 4, 'retweet_count': 1}, 'is_translatable': False, 'views': {'count': '28262', 'state': 'EnabledWithCount'}, 'source': '<a href="https://mobile.twitter.com" rel="nofollow">Twitter Web App</a>', 'note_tweet': {'is_expandable': True, 'note_tweet_results': {'result': {'id': 'Tm90ZVR3ZWV0OjE3OTQyOTEyMzEyODA3NjY5NzY=', 'text': "Same view here.\n\nBecause $ETH didn't move straight away people assume everything was priced in and/or the ETF isn't bullish for some reason.\n\nETH went to $3950 on the day of the approval and this was paired with a lot of uncertainty at the time. Logically speaking it wouldn't make much sense for price not to re-visit that level after approval which gave a lot more certainty. \n\nNot just the ETH approval itself made it bullish but also the massive pivot and tone of the overall stance against the crypto sector by the US government. Which I'd say is arguably more bullish for the sector as a whole than the ETH ETF itself.\n\nI think it's more likely that a ton of longs from the initial impulse on Monday still have been unwinding and being flushed out causing the momentum to stall.\n\nMarket participants are aware of this and therefore have had no desire to start allocating straight away. Especially TradFi, which is a lot slower than the crypto natives.\n\nSo by the time the big guys start bidding, the zoomer minds of the crypto natives have already moved on because it didn't pump 50% the same day while the next best dog coin did.\n\nKeep in mind, $ETH is still up +22% on the week. And yes, I'd argue that this isn't massive for all that we've achieved but I think we should give it some more time as well.\n\nThe #Bitcoin ETF also ended up not being priced in properly at all. Even though it rallied from $25K to $49K on the back of the pricing in to the approval. Yet, 2 months later it traded 50% higher due to all the ETF inflows. And this is with a relatively big correction in between.\n\nAnyways, time will tell. Just think people are a bit quick to jump to conclusions without giving this some time to play out.", 'entity_set': {'hashtags': [{'indices': [1317, 1325], 'text': 'Bitcoin'}], 'symbols': [{'indices': [25, 29], 'text': 'ETH'}, {'indices': [1152, 1156], 'text': 'ETH'}], 'urls': [], 'user_mentions': []}, 'richtext': {'richtext_tags': []}, 'media': {'inline_media': []}}}}, 'quoted_status_result': {'result': {'__typename': 'Tweet', 'rest_id': '1794287313008083112', 'core': {'user_results': {'result': {'__typename': 'User', 'id': 'VXNlcjozNDA1NzQzMzUz', 'rest_id': '3405743353', 'affiliates_highlighted_label': {}, 'has_graduated_access': True, 'is_blue_verified': True, 'profile_image_shape': 'Circle', 'legacy': {'can_dm': True, 'can_media_tag': False, 'created_at': 'Mon Aug 31 16:57:11 +0000 2015', 'default_profile': True, 'default_profile_image': False, 'description': 'Ex Siemens Engineer turned Full time trader\nNarrative Submitooor\nProfessional Shitposter', 'entities': {'description': {'urls': []}}, 'fast_followers_count': 0, 'favourites_count': 55526, 'followers_count': 25254, 'friends_count': 1224, 'has_custom_timelines': True, 'is_translator': False, 'listed_count': 735, 'location': '', 'media_count': 2964, 'name': 'Reetika', 'normal_followers_count': 25254, 'pinned_tweet_ids_str': [], 'possibly_sensitive': False, 'profile_image_url_https': 'https://pbs.twimg.com/profile_images/1756678255011110914/3DqFpjkI_normal.jpg', 'profile_interstitial_type': '', 'screen_name': 'ReetikaTrades', 'statuses_count': 15157, 'translator_type': 'none', 'verified': False, 'want_retweets': False, 'withheld_in_countries': []}, 'professional': {'rest_id': '1511606716194070529', 'professional_type': 'Creator', 'category': []}}}}, 'unmention_data': {}, 'edit_control': {'edit_tweet_ids': ['1794287313008083112'], 'editable_until_msecs': '1716630006000', 'is_edit_eligible': True, 'edits_remaining': '5'}, 'is_translatable': False, 'views': {'count': '47880', 'state': 'EnabledWithCount'}, 'source': '<a href="http://twitter.com/download/iphone" rel="nofollow">Twitter for iPhone</a>', 'note_tweet': {'is_expandable': True, 'note_tweet_results': {'result': {'id': 'Tm90ZVR3ZWV0OjE3OTQyODczMTI5NDEwMDY4NDk=', 'text': 'People were max long into a high volatility event expecting up only instantly and are now bearish after a pullback cause they’re liquidated\n\nA lil bit of chop then up seems like the most likely path \n\nDon’t midcurve the long term implications of Larry Fink shilling ETH etf to everyone', 'entity_set': {'hashtags': [], 'symbols': [], 'urls': [], 'user_mentions': []}}}}, 'quotedRefResult': {'result': {'__typename': 'Tweet', 'rest_id': '1793250890356338975'}}, 'legacy': {'bookmark_count': 5, 'bookmarked': False, 'created_at': 'Sat May 25 08:40:06 +0000 2024', 'conversation_id_str': '1794287313008083112', 'display_text_range': [0, 276], 'entities': {'hashtags': [], 'symbols': [], 'timestamps': [], 'urls': [], 'user_mentions': []}, 'favorite_count': 132, 'favorited': False, 'full_text': 'People were max long into a high volatility event expecting up only instantly and are now bearish after a pullback cause they’re liquidated\n\nA lil bit of chop then up seems like the most likely path \n\nDon’t midcurve the long term implications of Larry Fink shilling ETH etf to', 'is_quote_status': True, 'lang': 'en', 'quote_count': 3, 'quoted_status_id_str': '1793250890356338975', 'quoted_status_permalink': {'url': 'https://t.co/hISaLBFArv', 'expanded': 'https://twitter.com/reetikatrades/status/1793250890356338975', 'display': 'x.com/reetikatrades/…'}, 'reply_count': 19, 'retweet_count': 9, 'retweeted': False, 'user_id_str': '3405743353', 'id_str': '1794287313008083112'}}}, 'legacy': {'bookmark_count': 8, 'bookmarked': False, 'created_at': 'Sat May 25 08:55:40 +0000 2024', 'conversation_id_str': '1794291231473750116', 'display_text_range': [0, 276], 'entities': {'hashtags': [], 'symbols': [{'indices': [25, 29], 'text': 'ETH'}], 'timestamps': [], 'urls': [], 'user_mentions': []}, 'favorite_count': 165, 'favorited': False, 'full_text': "Same view here.\n\nBecause $ETH didn't move straight away people assume everything was priced in and/or the ETF isn't bullish for some reason.\n\nETH went to $3950 on the day of the approval and this was paired with a lot of uncertainty at the time. Logically speaking it wouldn't", 'is_quote_status': True, 'lang': 'en', 'quote_count': 0, 'quoted_status_id_str': '1794287313008083112', 'quoted_status_permalink': {'url': 'https://t.co/J97NkSvZoe', 'expanded': 'https://twitter.com/ReetikaTrades/status/1794287313008083112', 'display': 'x.com/ReetikaTrades/…'}, 'reply_count': 36, 'retweet_count': 16, 'retweeted': False, 'user_id_str': '918138253617790976', 'id_str': '1794291231473750116'}}}, 'tweetDisplayType': 'Tweet'}, 'clientEventInfo': {'component': 'following_in_network', 'element': 'tweet', 'details': {'timelinesDetails': {'injectionType': 'FollowingInNetwork', 'controllerData': 'DAACDAABDAABCgABAEAABAoCAAEKAAKAAAAAAAEhAAoACbpScy0XTnWhCAALAAAAYw8ADAMAAAAQAQACCgQAQAAAIQEAAAAAgAoADrElPs+500r/AAAAAA=='}}}}
 ```

![image](https://github.com/StephanAkkerman/fintwit-bot/assets/45365128/4c601862-62cb-40ed-bfd1-25e4e0c1d9eb)

https://twitter.com/DaanCrypto/status/1794291231473750116

**Comment by StephanAkkerman** [comment]
Fixed with 3ebe12f
